Cargo.toml

```diff
@@ -1,5 +1,11 @@
 [workspace]
-members = ["crates/*"]
+members = [
+  "crates/cli",
+  "crates/ffi",
+  "crates/kotlin-ffi",
+  "crates/yttrium",
+  "crates/yttrium_dart/rust",
+]
```

Comment:
```suggestion
members = [
  "crates/*",
  "crates/yttrium_dart/rust",
]
```

Reply:
Nevermind I see now why you needed this @quetool

Reply:
Ah LOL so should I revert it?